File: Data Structures & Algorithms/remove-element/submission-9.cs
public class Solution {
    public int RemoveElement(int[] nums, int val) {
        int low = 0;
        for (int i = 0; i < nums.Length; ++i) {
            if (nums[i] != val) {
                nums[low] = nums[i];
                ++low;
            }   
        }
        return low;
    }
}

File: Data Structures & Algorithms/monotonic-array/submission-9.py
class Solution:
    def isMonotonic(self, nums: List[int]) -> bool:
        increasing = False
        decreasing = False

        for i in range(0, len(nums) - 1):
            if nums[i + 1] > nums[i]:
                increasing = True
            elif nums[i + 1] < nums[i]:
                decreasing = True
            if increasing and decreasing:
                return False
        return not(increasing and decreasing)